sale_comment_template/migrations/8.0.1.0.0/pre-migration.py

```diff
@@ -0,0 +1,15 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Tecnativa - Vicent Cubells
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openupgradelib import openupgrade
+
+
+column_renames = {
+    'sale_order': [('condition_template_id', None)],
+}
+
+
+@openupgrade.migrate()
+def migrate(cr, version):
+    openupgrade.rename_columns(cr, column_renames)
```

Comment:
Do we really need to rename column? Remember that SQL columns are not dropped, so the only reason for renaming is because there's another column with the same name with other function or other values.